.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -175,6 +175,14 @@ AddType text/vtt                            vtt
     RewriteRule ^container-platform/(4\.11|4\.12|4\.13|4\.14)/logging/config/cluster-logging-moving-nodes.html /container-platform/$1/logging/scheduling_resources/logging-node-selectors.html [NE,R=302]
     RewriteRule ^(rosa|dedicated)/logging/config/cluster-logging-moving-nodes.html /$1/logging/scheduling_resources/logging-node-selectors.html [NE,R=302]
 
+    # Redirects for observability rework per https://github.com/openshift/openshift-docs/pull/71248
+    RewriteRule ^container-platform/4.14/power_monitoring/installing-power-monitoring.html /container-platform/latest/observability/power_monitoring/installing-power-monitoring.html [NE,R=302]
+    RewriteRule ^container-platform/4.14/power_monitoring/power-monitoring-release-notes.html /container-platform/4.14/observability/power_monitoring/power-monitoring-release-notes.html [NE,R=302]
+    RewriteRule ^container-platform/4.14/power_monitoring/power-monitoring-overview.html /container-platform/4.14/observability/power_monitoring/power-monitoring-overview.html [NE,R=302]
+    RewriteRule ^container-platform/4.14/power_monitoring/configuring-power-monitoring.html /container-platform/4.14/observability/power_monitoring/configuring-power-monitoring.html [NE,R=302]
+    RewriteRule ^container-platform/4.14/power_monitoring/visualizing-power-monitoring-metrics.html /container-platform/4.14/observability/power_monitoring/visualizing-power-monitoring-metrics.html [NE,R=302]
+    RewriteRule ^container-platform/4.14/power_monitoring/uninstalling-power-monitoring.html /container-platform/4.14/observability/power_monitoring/uninstalling-power-monitoring.html [NE,R=302]
+
```

Comment:
**praise:** 👏 Thank you for creating neat redirects. LGTM.

**thought:** However, I see that the whole section is moving so we can reduce it to the following:
```
RewriteRule ^container-platform/4.14/power_monitoring/installing-power-monitoring.html /container-platform/latest/observability/power_monitoring/installing-power-monitoring.html [NE,R=302]
RewriteRule ^container-platform/4\.14/power_monitoring/(.*)$ /container-platform/4.14/observability/power_monitoring/$1 [NE,R=302,L]
```
- For the first link it will match the first rule always and the next one for all others https://htaccess.madewithlove.com?share=3839cd11-b247-4606-9ab8-803309a38cbe

Reply:
Thank you Gaurav, good idea. I've compressed them down to 1 rule since Lindsey confirmed they can all be 4.14.